github/ci/prow-deploy/molecule/default/Dockerfile.j2

```
@@ -1,41 +1,39 @@
 FROM {{ item.image }}
 
 RUN printf '#!/bin/bash\n\
-echo "Docker in Docker enabled, initializing..."\n\
+echo "Podman in Container enabled, initializing..."\n\
 printf '=%.0s' {1..80}; echo\n\
-# If we have opted in to docker in docker, start the docker daemon,\n\
 (\n\
-    if [ -f "/etc/default/docker" ]; then\n\
-        source /etc/default/docker\n\
-    fi\n\
-    /usr/bin/dockerd \
-        -p /var/run/docker.pid \
-        --data-root=/docker-graph \
-        --init-path /usr/libexec/docker/docker-init \
-        --userland-proxy-path /usr/libexec/docker/docker-proxy \
-        ${DOCKER_OPTS} \
-            >/var/log/dockerd.log 2>&1 &\n\
+    PODMAN_SOCKET_PATH=/run/podman\n\
+    PODMAN_SOCKET=${PODMAN_SOCKET_PATH}/podman.sock\n\
+    mkdir -p ${PODMAN_SOCKET_PATH}\n\
+    podman system service \
+                -t 0 \
+                unix://${PODMAN_SOCKET} \
+                >/var/log/podman.log 2>&1 & \
+        echo "${!}" > /var/run/podman.pid \n\
```

nit: indentation seems off
```suggestion
    echo "${!}" > /var/run/podman.pid \n\
```

---

It should be updated now.